packages/main/src/plugin/container-registry.ts

```diff
@@ -476,7 +475,7 @@ export class ContainerProviderRegistry {
                 Created: moment(podmanContainer.Created).unix(),
                 State: podmanContainer.State,
                 StartedAt,
-                Command: podmanContainer.Command?.length > 0 ? podmanContainer.Command[0] : undefined,
+                Command: podmanContainer.Command,
```

Comment:
if the null is coming from here we should try to avoid it, and use a nullish coalescing operator
```suggestion
                Command: podmanContainer.Command ?? undefined,
```